src/Majora/Framework/Serializer/MajoraSerializer.php

```diff
@@ -7,8 +7,7 @@
 /**
  * Base class for fixtures repository.
  */
-class MajoraSerializer
-    implements SerializerInterface
+class MajoraSerializer implements SerializerInterface
 {
     protected $handlers;
```

Comment:
Ca serait cool un phpdoc là-dessus, pour avoir la complétion sur `$this->handlers[$format]->serialize` plus bas